src/messages/MStatfs.h

```diff
@@ -22,10 +22,12 @@
 class MStatfs : public PaxosServiceMessage {
 public:
   uuid_d fsid;
+  boost::optional<int> data_pool;
 
-  MStatfs() : PaxosServiceMessage(CEPH_MSG_STATFS, 0) {}
-  MStatfs(const uuid_d& f, ceph_tid_t t, version_t v) :
-    PaxosServiceMessage(CEPH_MSG_STATFS, v), fsid(f) {
+  MStatfs() : PaxosServiceMessage(CEPH_MSG_STATFS, 0, 2) {}
```

Comment:
should use the const definitions at the top of the class like other messages... see e.g.
https://github.com/ceph/ceph/blob/master/src/messages/MOSDOp.h#L36